keras_trainer/trainer.py

```diff
@@ -91,10 +96,16 @@ def __init__(self, **options):
             'options': options
         }
 
+        if isinstance(self.loss_function, list) and len(self.loss_function) > 1:
+            self.n_outputs = len(self.loss_function)
```

Comment:
The number of output labels that we will generate. 
The number of model outputs is defined by a custom model.

Reply:
is the "number of output labels" the same as the number of classes? I don't see the relationship between the number of loss functions and the number of output labels?

Reply:
Means the number of times you have to return `batch_y` to be able to run it in  the keras `fit_generator` function. 
We do `[batch_y] * n_outputs` where n_outputs is the len(loss_functions). Because they require one label batch per output.

Reply:
There is an example in the notebook. I added that parameter to handle it internally. It will only work if your loss functions take the same output labels though.